packages/errors/src/errors.ts

```diff
@@ -1057,6 +1057,12 @@ export const AllCypressErrors = {
 
         You can safely remove this option from your config.`
   },
+  EXPERIMENTAL_STUDIO_REMOVED: () => {
+    return errTemplate`\
+        We're ending the experimental phase of Cypress Studio and have learned a lot. Stay tuned for updates on Studio's official release in the future. You can leave feedback here: http://on.cypress.io/studio-beta.
```

Comment:
@davidmunechika Is this link working when you click on it? In GitHub it's adding the period to the url so the link doesn't work, but this may not be a problem in the actual product.

Reply:
<img width="574" alt="Screenshot 2022-03-09 at 12 07 08 PM" src="https://user-images.githubusercontent.com/27690333/157493486-dd7f51ee-e156-44e7-b07b-48b1c616b228.png">

Yes, this works for me in the app. It separates the link from the period.